frontend/.env.development

```diff
@@ -5,4 +5,5 @@ NODE_ENV=development
 ENDPOINT_PROD=url
 ENDPOINT_DEV=localhost
 
+ENDPOINT_USER_PORT=5000
```

Comment:
We can create another PR for changing the user service port number, based on @tlyi feedback to me last time, Mac uses port 5000 for something else, so we can't use this port. A suggestion is to switch the port to 5005 instead. Do you think we should change the port in this PR or another small PR?

Reply:
i think just change here bah